vuepress/docs/next/docs/reference/convert-bundle.md

```diff
@@ -22,7 +24,42 @@ The improvements include:
 
 * [Bundle Version Comparison Table](./bundle-comparison.md)
 
-## Upgrade Process
+## Automated Process
+
+Use the following ent bundle command to convert your v1 bundle, using the path to the source descriptor YAML. The new bundle will be located in the same parent directory as the original and noted when the execution is complete. The interactive process will prompt you for some specifications about your bundle. These details are below. 
+
+``` bash 
+ent bundle convert --bundle-path /YOUR-BUNDLE-PATH
+```
+
+* For every micro frontend in your bundle, you will be asked to confirm if they should be treated as MFEs, widgets, or configuration widgets. 
+
+* To provide the path containing the Docker files for services, you can enter a location when promted or use the flag `--svc-path`, e.g.,  `ent bundle convert --bundle-path YOUR-BUNDLE-PATH --svc-path YOUR-PATH/YOUR-SERVICES-FOLDER`.
+
+* This automated process preserves these properties: healthCheckPath, roles, dbms, permissions, securityLevel, resources, ingressPath.
+
+* Static assets or resources are moved to the `platform` folder in the v5 project. The typical assets included in this conversion process are as follows, in their corresponding folders. If any assets are included under names not on this list, they must be dealt with manually. 
+PLATFORM_FOLDERS_NAME = [
+'fragments',
+'categories',
+'pages',
+'pageTemplates',
+'contentTypes',
+'contentTemplates',
+'contents',
+'assets',
+'groups',
+'labels',
+'languages',
+'pageModels',
```

Comment:
pageTemplates is the same as the older pageModels, and contentTemplates is the same as contentModels. Maybe the deprecated model names could be added as a note rather than mixed in here whether the duplication might be confusing?

Reply:
@nshaw if the older v1 bundle had a contentModel folder, will it be converted to contentTemplates folder or stay in the same contentModel folder?